### PR TITLE
Docs 3080 private key jwt updates

### PR DESCRIPTION
**Pull request by avanscoy**

### CODE REVIEW

File: articles/api/authentication/_multifactor-authentication.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ If OTP is supported by the user and you don't want to request a different factor
 | Parameter        | Description |
 |:-----------------|:------------|
 | `mfa_token` <br/><span class="label label-danger">Required</span> | The token received from `mfa_required` error. |
+| `client_assertion`| A JWT containing a signed assertion with your application credentials. Required when Private Key JWT is your application authentication method.|
+|`client_assertion_type`| The value is `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`.  Required when Private Key JWT is the application authentication method.|
```
Comment (CharlesRea) [inline articles/api/authentication/_multifactor-authentication.md]: Should we bold "Required" and "Application Authentication Method" to be consistent with `client_secret` below? Or remove the bold text from them?

Can we switch the ordering so these below `client_id` in the table? I think it feels most natural to have `client_id` first (very minor point though!). And the same for the similar exmples below.

**Review by CharlesRea — COMMENTED** [review]
Looks good!

File: articles/api/authentication/_multifactor-authentication.md (hunk 1 above)
Comment (avanscoy) [reply articles/api/authentication/_multifactor-authentication.md]: Sure! Is the preference bold or not? I feel like that could be too many bold statements for a short list, but lmk what you think.

I can switch the `client_id` param, no problem.

Comment (CharlesRea) [reply articles/api/authentication/_multifactor-authentication.md]: I'd agree, feels like too much bold. Should we get rid of it altogether?

**Review by pdillon — APPROVED** [review]

**Pull request merged**